posthog/api/person.py

```diff
@@ -98,15 +98,48 @@ def to_representation(self, instance: Person) -> Dict[str, Any]:
 
 class PersonFilter(filters.FilterSet):
     email = filters.CharFilter(field_name="properties__email")
-    distinct_id = filters.CharFilter(field_name="persondistinctid__distinct_id")
-    key_identifier = filters.CharFilter(method="key_identifier_filter")
+    distinct_id = filters.CharFilter(method="distinct_id_filter")
+    key_identifier = filters.CharFilter(method="key_identifier_filter", help_text="Filter on email or distinct ID")
+    uuid = filters.CharFilter(method="uuid_filter")
+    search = filters.CharFilter(method="search_filter")
+    cohort = filters.CharFilter(field_name="cohort__id", help_text="ID of a cohort the user belongs to")
+    properties = filters.CharFilter(method="properties_filter")
+
+    def __init__(self, data=None, queryset=None, *, request=None, prefix=None, team_id=None):
+        self.team_id = team_id
+        return super().__init__(data=data, queryset=queryset, request=request, prefix=prefix)
+
+    def distinct_id_filter(self, queryset, attr, *args, **kwargs):
```

Comment:
Does changing the signature to 

```suggestion
    def distinct_id_filter(self, queryset, attr, distinct_id, *args, **kwargs):
```

Work i.e. giving distinct_id an explicit var name rather than `args[0]` so it's more obvious what it is.